crates/napi/src/next_api/endpoint.rs

```diff
@@ -146,7 +146,7 @@ pub async fn endpoint_write_to_disk(
 
             Ok((written.clone(), issues.clone(), diagnostics.clone()))
         })
-        .or_else(|e| ctx.throw_turbopack_internal_result(&e))
+        .or_else(|e| ctx.throw_turbopack_internal_result(&e.into()))
```

Comment:
IDK if there are other callsites, but if `throw_turbopack_internal_result` now only receives `TurboTasksExecutionError`, it could be modified to accept `TurboTasksExecutionError` or `impl Error` (which sadly `anyhow::Error` does/can not implement).